### PR TITLE
feat: implement confirm anagkazo banking

### PR DESCRIPTION
**Pull request by ahadziii**

### CODE REVIEW

File: api/src/resolvers/anagkazo/treasury-resolvers.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const treasuryMutations = {
         await session.run(cypher.checkIfConfirmed, args)
       )
 
-      if (checkAlreadyConfirmed) {
+      if (checkAlreadyConfirmed=== true) {
```
Comment (jaedag) [inline api/src/resolvers/anagkazo/treasury-resolvers.js:37]: This looks like unnecessary code added

File: web-react-ts/src/pages/services/ServicesMenu.tsx [hunk 2]
```diff
@@ -101,19 +101,26 @@ const Services = () => {
...
+          <MenuButton
+            iconComponent={CashCoin}
+            title="Midweek Banking"
+            color="banking"
+            onClick={() =>
+              navigate(`/services/${churchType.toLowerCase()}/midweek-banking`)
+            }
+            noCaption
+          />
+          {/* remove this when you are done */}
```
Comment (jaedag) [inline web-react-ts/src/pages/services/ServicesMenu.tsx:113]: What is this comment about?

File: web-react-ts/src/pages/services/defaulters/DefaultersQueries.ts [hunk 3]
```diff
@@ -522,6 +522,7 @@ export const STREAM_BANKING_DEFAULTERS_LIST = gql`
           fullName
           phoneNumber
           whatsappNumber
+          pictureUrl
         }
         bacenta {
```
Comment (jaedag) [inline web-react-ts/src/pages/services/defaulters/DefaultersQueries.ts:527]: Maybe we should delete this object if it's not necessary.

File: web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+type Defaulter = {
+  bacenta: {}
+  id: string
+  leader: { pictureUrl: string; fullName: string }
+  meetingDay: { day: string }
+  name: string
+  services: [{}]
+  __typename: string
+}
```
Comment (jaedag) [inline web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx:33]: Will have to review this type again later. Not sure about the bacenta dad services and __typename

Typename should have been ChurchLevel

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx [hunk 5]
```diff
@@ -23,13 +23,9 @@ type FormOptions = {
 }
 
 type Defaulter = {
-  bacenta: {}
-  id: string
   leader: { pictureUrl: string; fullName: string }
-  meetingDay: { day: string }
   name: string
   services: [{}]
-  __typename: string
```
Comment (jaedag) [inline web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx]: type of typename should be ChurchLevel

Comment (jaedag) [reply web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx]: Type of Bacenta should also be something else. NOt this empty object.

File: web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx [hunk 6]
```diff
@@ -23,13 +23,9 @@ type FormOptions = {
 }
 
 type Defaulter = {
-  bacenta: {}
-  id: string
   leader: { pictureUrl: string; fullName: string }
-  meetingDay: { day: string }
```
Comment (jaedag) [inline web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx]: Type of meeting day should be one of the days of the week. Not string.

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx (hunk 5 above)
Comment (ahadziii) [reply web-react-ts/src/pages/services/banking/anagkazo/ConfirmAnagkazoBanking.tsx]: I have removed that code

File: api/src/resolvers/anagkazo/treasury-resolvers.ts [hunk 7]
```diff
@@ -39,7 +39,7 @@ const treasuryMutations = {
         await session.run(anagkazo.checkIfConfirmed, args)
       )
 
-      if (checkAlreadyConfirmed=== true) {
+      if (checkAlreadyConfirmed === true) {
```
Comment (jaedag) [inline api/src/resolvers/anagkazo/treasury-resolvers.ts:42]: Redundant Code

**Review by jaedag — APPROVED** [review]

**Review by dabick14 — APPROVED** [review]

File: api/src/resolvers/anagkazo/treasury-cypher.ts [hunk 8]
```diff
@@ -8,8 +8,11 @@ const anagkazo = {
     `,
   checkIfConfirmed: `
```
Comment (jaedag) [inline api/src/resolvers/anagkazo/treasury-cypher.ts:9]: How many DB Hits

**Review by jaedag — APPROVED** [review]

File: api/src/resolvers/anagkazo/treasury-cypher.ts (hunk 8 above)
Comment (ahadziii) [reply api/src/resolvers/anagkazo/treasury-cypher.ts:9]: @dabick14 Question for you

Comment (dabick14) [reply api/src/resolvers/anagkazo/treasury-cypher.ts:9]: 3 hits

**Review by jaedag — APPROVED** [review]

**Pull request closed without merging**